cloudbridge/cloud/providers/gce/resources.py

```diff
@@ -176,66 +177,80 @@ def zones(self):
 
 
 class GCEFirewallsDelegate(object):
+    NETWORK_URL_PREFIX = 'global/networks/'
```

Comment:
NETWORK_URL_PREFIX is class private and so I prefixed it with an underscore. DEFAULT_NETWORK is used in the GCESecurityGroup class, too.